zebra/zebra_mpls.c

```diff
@@ -2749,30 +2822,54 @@ int mpls_lsp_uninstall(struct zebra_vrf *zvrf, enum lsp_types_t type,
 	return 0;
 }
 
+int mpls_lsp_uninstall_all_vrf(struct zebra_vrf *zvrf, enum lsp_types_t type,
```

Comment:
Add comment to precise the purpose of this new function

Reply:
This function is already documented in the header file (_zebra/zebra_mpls.h_).

Reply:
OK. Don't know where is the best place to put comment. We'll need to decide globally in FRR to add some coding rules about comment and Doxygen like description of non static functions to auto-generate the doc.